README.md

```diff
@@ -1,49 +1,177 @@
+- And more... see [our Thanks page][url-thanks]
+
+All contributions must respect the [Code of Conduct][file-conduct].
+
+### Credits
+
+<details><summary>Direct contributions</summary>
+
+- [TheYTG123](https://www.curseforge.com/members/theytg123/projects) on CurseForge who helped me with Jumploader to release the 1.16.3 version a little faster
+- [triphora](https://github.com/triphora) who suggested and helped with the Modrinth format and formerly moderated FO Discord server
+- [RozeFound](https://github.com/RozeFound) for helping with Python scripts and providing lots of constructive feedback for the pack
+- [TheMadHau5](https://github.com/themadhau5), [redbrain](https://github.com/redbrain), [Kichura](https://github.com/Kichura) for the [fancy mod table](https://github.com/Fabulously-Optimized/fabulously-optimized/blob/main/INCLUDED-MODS.md)
+- [RaptaG](https://github.com/RaptaG) and [Ultrasonic1209](https://github.com/Ultrasonic1209) for the [Packwiz mod ignoring tutorials](https://fabulously-optimized.gitbook.io/modpack/readme/multimc-auto-update#can-i-ignore-some-of-the-mods)
+- Other people creating pull requests for [the main project](https://github.com/Fabulously-Optimized/fabulously-optimized/graphs/contributors) and [the wiki](https://github.com/Fabulously-Optimized/wiki/graphs/contributors)
+- [People who have created videos about the modpack](https://github.com/Fabulously-Optimized/fabulously-optimized#reviews)
+- Everyone who has suggested ideas and reported bugs!
+
+</details>
+
+<details><summary>Special thanks</summary>
+
+- [jellysquid3](https://github.com/jellysquid3) for creating [Sodium](https://www.curseforge.com/minecraft/mc-mods/sodium), [Lithium](https://www.curseforge.com/minecraft/mc-mods/lithium), [Phosphor](https://www.curseforge.com/minecraft/mc-mods/phosphor) which were my initial inspiration for creating the modpack
+- [LambdAurora](https://github.com/LambdAurora) for creating [the list of OptiFine alternatives](https://lambdaurora.dev/optifine_alternatives) which the pack does get some mod ideas from, [LambDynamicLights](https://www.curseforge.com/minecraft/mc-mods/lambdynamiclights) and [LambdaBetterGrass](https://www.curseforge.com/minecraft/mc-mods/lambdabettergrass)
+- [comp500](https://github.com/comp500) for creating [Jumploader](https://www.curseforge.com/minecraft/mc-mods/jumploader) - a mod that made the modpack possible in Minecraft 1.16.x, [Indium](https://modrinth.com/mod/indium) - mod that provided the rendering API for Sodium for increased mod compatibility and [packwiz](https://github.com/comp500/packwiz) - a tool that makes the MultiMC auto-update and in the future, vanilla installer happen
+- Mod devs that have joined my Discord and are discussing the modpack's development there
+- All developers who made the mods that are, have been and will be in the modpack
+- Everyone who uses, tests and shares the modpack!
+
+</details>
+
+<!-- Files -->
+<!-- ( ): should be in the wiki -->
```

Comment:
- What is `( ):`? 
- I have considered putting the mod list to wiki, but in its current format it just doesn't horizontally fit very well. If you can think of an alternative design and PR there, sure.
- Code of conduct can be in the wiki, but it feels more relevant to be in the repo... I don't know.

Reply:
- It's TODO lol
- Wouldn't a message like the one [here](<https://wiki.download.fo/readme/changed-options#options>) suffice?
- Yes, that's true, since the repo hosts the code

Reply:
> Wouldn't a message like the one [here](https://wiki.download.fo/readme/changed-options#options) suffice?

Even that has the last (the most important) column hidden by default, which I don't like at all. Need to find a more usable design for that...

> What I meant is that the Wiki should link to the Code of Conduct in the "Thanks!" page

Could be done, yeah. I'll make an issue for myself.

Reply:
I guess I'll leave the `INCLUDED-MODS.md` question for another time...
This PR already has enough changes as it is :upside_down_face: